src/client/components/mapComponents/Icons.js

```diff
@@ -0,0 +1,12 @@
+import { Icon } from 'leaflet';
+
+export default {
+  TreasureIcon: new Icon({
+    iconUrl: '/src/client/assets/images/treasure.png',
+    iconSize: [71, 71],
+  }),
+  DeafultMapMarker: new Icon({
```

Comment:
'DeafultMapMarker' spelling may be changed to 'DefaultMapMarker'

Reply:
oops! you are right :)